.gitignore

```diff
@@ -107,3 +107,5 @@ node_modules/
 
 # IDEs config
 .vscode/
+
+services/docker-compose.override.yml
```

Comment:
why this? is this generated?

Reply:
I use this trick to shorten my docker-compose commands while developing. E.g. if I want to build development images, one would need to call
``` docker-compose -f docker-compose.yml -f docker-compose.development.yml build ```
but instead I copy and rename ``docker-compose.development.yml `` as ''docker-compose.override.yml '' and then I can simplify the call to
``` docker-compose build ```
Check this [doc](https://docs.docker.com/compose/extends/#multiple-compose-files)